cloudinit/cmd/main.py

```diff
@@ -42,6 +38,10 @@
 from cloudinit.config import cc_set_hostname
 from cloudinit import dhclient_hook
 
+from cloudinit import patcher
+patcher.patch_logging()
+from cloudinit import log as logging  # noqa: E402
+
```

Comment:
Let's not move this. I want the patch to happen before other cloud-init imports.

Reply:
Okay, understood. My question is - can we do it in some other place and avoid this warning?

Like in `/usr/bin/cloud-init` before anything starts?